src/lib/getVisibilityWatcher.ts

```diff
@@ -93,13 +95,15 @@ export const getVisibilityWatcher = () => {
       // https://bugs.chromium.org/p/chromium/issues/detail?id=1133363
       setTimeout(() => {
         firstHiddenTime = initHiddenTime();
-        addChangeListeners();
       });
     });
   }
   return {
     get firstHiddenTime() {
       return firstHiddenTime;
     },
+    set onHidden(cb: () => void) {
```

Comment:
```suggestion
    onHidden(cb: () => void) {
```
I think you want this to be a normal function and not a setter, right?